### PR TITLE
setup BED theming  and navigation drawer

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+VUE_APP_MAPBOX_TOKEN='pk.eyJ1IjoiZ2xvYmFsLWRhdGEtdmlld2VyIiwiYSI6ImNqdG9lYWQ3NTFsNWk0M3Fqb2Q5dXBpeWUifQ.3DvxuGByM33VNa59rDogWw'
```
Comment (SiggyF) [inline .env]: keep `.env` local

Comment (CindyvdVries) [reply .env]: Woops, thought I added this to the gitignore.

Check

File: package.json [hunk 2]
```diff
@@ -10,9 +10,12 @@
   },
   "dependencies": {
     "core-js": "^3.6.5",
+    "mapbox-gl": "^1.11.1",
     "register-service-worker": "^1.7.1",
     "vue": "^2.6.11",
+    "vue-html-loader": "^1.2.4",
```
Comment (SiggyF) [inline package.json:16]: is this needed?

File: src/components/CustomIcon.vue [hunk 3]
```diff
@@ -0,0 +1,59 @@
+<template>
+  <span
```
Comment (SiggyF) [inline src/components/CustomIcon.vue:2]: See https://vuetifyjs.com/en/customization/icons/#using-custom-icons .

File: src/components/MapComponent.vue [hunk 4]
```diff
@@ -0,0 +1,38 @@
+<template>
```
Comment (SiggyF) [inline src/components/MapComponent.vue:1]: Is this component needed? It does not do anything. Could you also move v-mapbox to the view?

File: src/components/SideMenu.vue [hunk 5]
```diff
@@ -0,0 +1,75 @@
+<template>
+  <v-card>
```
Comment (SiggyF) [inline src/components/SideMenu.vue:2]: Should card be in drawer, or drawer in card?

File: src/css/main.css [hunk 6]
```diff
@@ -0,0 +1,3 @@
+html {
```
Comment (SiggyF) [inline src/css/main.css:1]: Should this be on html?

File: src/css/typography.css [hunk 7]
```diff
@@ -0,0 +1,19 @@
+@font-face {
+  font-family: 'sansa_con_probold';
+  src: url('/fonts/sansapro-bold-webfont.woff2') format('woff2'),
+       url('/fonts/sansapro-bold-webfont.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+html {
```
Comment (SiggyF) [inline src/css/typography.css:10]: should these be defined on html?

File: vue.config.js [hunk 8]
```diff
@@ -1,5 +1,16 @@
 module.exports = {
-  transpileDependencies: [
-    'vuetify'
-  ]
+  transpileDependencies: ['vuetify'],
```
Comment (SiggyF) [inline vue.config.js:2]: Test the loading of components?

**Review by SiggyF — APPROVED** [review]
Some questions

**Pull request merged**